src/node_crypto.cc

```diff
@@ -5572,6 +5591,65 @@ void RandomBytes(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void RandomBytesBuffer(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  if (!Buffer::HasInstance(args[0])) {
+    return env->ThrowTypeError("buf must be a Buffer");
```

Comment:
I know this is only copied, but you can write this as `args[0]->IsUint8Array()`, too; that’s what `Buffer::HasInstance` does under the hood.